zerver/models.py

```diff
@@ -1694,6 +1694,7 @@ class RealmAuditLog(models.Model):
     acting_user = models.ForeignKey(UserProfile, null=True, related_name='+', on_delete=CASCADE)  # type: Optional[UserProfile]
     modified_user = models.ForeignKey(UserProfile, null=True, related_name='+', on_delete=CASCADE)  # type: Optional[UserProfile]
     modified_stream = models.ForeignKey(Stream, null=True, on_delete=CASCADE)  # type: Optional[Stream]
+    modified_subscription = models.ForeignKey(Subscription, null=True, on_delete=CASCADE)  # type: Optional[Subscription]
```

Comment:
Do we need `modified_subscription`?  It seems like we could just have `modified_stream` and `modified_user`, and that pair allows one to look up the subscription object anyway...